task_12/app/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta http-equiv="Cache-Control" content="no-store">
+	<title>Pizzzzzzzza</title>
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<!-- <link href="https://use.fontawesome.com/releases/v5.0.4/css/all.css" rel="stylesheet"> -->
+</head>
+<body>
+	<header>
+		<div class=header__time>
+			<img src="images/001-square.png" alt="clock icon">
+			<time id="current-time"></time>
+		</div>
+		<div class="header__logo">
+			<img src="images/logo.png" alt="pizza-logo">
+		</div>
+		<div class="header__signup">
+			<button class="button button-signup">
+				<p>Login / Signup</p>
+			</button>
+		</div>
+	</header>
+	<main>
+		<button class="button button-add">
+			<p>Add new pizza</p>
```

Comment:
From [spec](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element):

> Content model: Phrasing content, but there must be no interactive content descendant.

> Phrasing content is the text of the document, as well as elements that mark up that text at the intra-paragraph level.

It means you cannot use `<p>` inside of `<button>`. Do you really need it?